test/sim/abilities/cudchew.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const assert = require('../../assert');
+const common = require('../../common');
+
+let battle;
+
+describe('Cud Chew', function () {
+	afterEach(() => battle.destroy());
+
+	it(`should eat berry a second time when taking residual damage`, function () {
+		battle = common.createBattle([
+			[{species: 'Tauros-Paldea-Combat', ability: 'cudchew', item: 'sitrusberry', moves: ['sleeptalk']}],
+			[{species: 'Ekans', moves: ['toxic']},  {species: 'Weezing-Galar', ability: 'neutralizinggas', moves: ['sleeptalk']}],
+		]);
```

Comment:
Was just copy and pasting from the other tests, but sure. Ekans was a bad example Poison-type I guess since Unnerve is one of its abilities, but there are _several_ tests already on PS with no ability set.